services/provider/server/server.go

```diff
@@ -1076,12 +1084,15 @@ func (s *OCSProviderServer) ReportStatus(ctx context.Context, req *pb.ReportStat
 		return nil, status.Errorf(codes.Internal, "Failed to get mirroring status for consumer.")
 	}
 
+	isNfsEnabled := storageCluster.Spec.NFS.Enable
```

Comment:
iirc Ohad mentioned we can always deploy NFS, possible to reconfirm?

Reply:
You mean enable NFS and deploy NFS CSI driver always or just NFS CSI drivers always?